### PR TITLE
[Core] Add CadIoModeler

### PR DESCRIPTION
**Pull request by tteschemacher**
Dear all,

this is a modeler wrapping the cad io input, to be able to use it within the novel modeler facilities.

### CODE REVIEW

File: kratos/modeler/cad_io_modeler.h [hunk 1]
```diff
@@ -0,0 +1,145 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//
```
Comment (loumalouomega) [inline kratos/modeler/cad_io_modeler.h:10]: You should add your name as author in the files

**Review by loumalouomega — COMMENTED** [review]
Minor comment

File: kratos/modeler/cad_io_modeler.h (hunk 1 above)
Comment (tteschemacher) [reply kratos/modeler/cad_io_modeler.h:10]: I use to avoid that due to bad experience ;)

**Comment by tteschemacher** [comment]
@pooyan-dadvand can we merge that? It just wraps the cad_json_input within a modeler.

**Review by pooyan-dadvand — APPROVED** [review]
👍

**Pull request merged**